bundle-workflow/src/s3_utility.py

```diff
@@ -0,0 +1,97 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
```

Comment:
You can untrack these files in your commit and still use them for local development. That way, they won't be raised as part of your PR.